packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -59,17 +59,16 @@ async function createDataObject<
 	// request mixin in
 	runtimeClass = mixinRequestHandler(
 		async (request: IRequest, runtimeArg: FluidDataStoreRuntime) => {
-			const maybeRouter: FluidObject<IFluidRouter> | undefined =
-				await runtimeArg.entryPoint?.get();
+			const maybeRouter: FluidObject<IFluidRouter> = await runtimeArg.entryPoint.get();
```

Comment:
```suggestion
			const router: FluidObject<IFluidRouter> = await runtimeArg.entryPoint.get();
```
and remove the next assert.